pyignite/queries/response.py

```diff
@@ -44,7 +45,7 @@ def __build_header(self):
                 ('query_id', ctypes.c_longlong),
             ]
 
-            if self.protocol_version and self.protocol_version >= (1, 4, 0):
+            if self.protocol_context and self.protocol_context.is_status_flags_supported():
```

Comment:
I suppose, that check for None is ambigous here.... Let's remove it (if we are here, protocol context have been already set)

Reply:
Done